bentoml/serve.py

```diff
@@ -75,56 +83,63 @@ def ensure_prometheus_dir(
     return alternative
 
 
-@inject
-def serve_development(
-    bento_identifier: str,
-    working_dir: str,
-    port: int = Provide[BentoMLContainer.api_server_config.port],
-    host: str = Provide[BentoMLContainer.api_server_config.host],
-    backlog: int = Provide[BentoMLContainer.api_server_config.backlog],
-    bentoml_home: str = Provide[BentoMLContainer.bentoml_home],
-    ssl_certfile: str | None = Provide[BentoMLContainer.api_server_config.ssl.certfile],
-    ssl_keyfile: str | None = Provide[BentoMLContainer.api_server_config.ssl.keyfile],
-    ssl_keyfile_password: str
-    | None = Provide[BentoMLContainer.api_server_config.ssl.keyfile_password],
-    ssl_version: int | None = Provide[BentoMLContainer.api_server_config.ssl.version],
-    ssl_cert_reqs: int
-    | None = Provide[BentoMLContainer.api_server_config.ssl.cert_reqs],
-    ssl_ca_certs: str | None = Provide[BentoMLContainer.api_server_config.ssl.ca_certs],
-    ssl_ciphers: str | None = Provide[BentoMLContainer.api_server_config.ssl.ciphers],
-    reload: bool = False,
-) -> None:
-    working_dir = os.path.realpath(os.path.expanduser(working_dir))
-    svc = load(bento_identifier, working_dir=working_dir)  # verify service loading
+def create_watcher(
+    name: str,
+    args: list[str],
+    *,
+    use_sockets: bool = True,
+    **kwargs: t.Any,
+) -> Watcher:
+    from circus.watcher import Watcher
+
+    return Watcher(
+        name=name,
+        cmd=sys.executable,
+        args=args,
+        copy_env=True,
+        stop_children=True,
+        use_sockets=use_sockets,
+        **kwargs,
+    )
 
-    from circus.sockets import CircusSocket  # type: ignore
-    from circus.watcher import Watcher  # type: ignore
 
-    prometheus_dir = ensure_prometheus_dir()
+def log_grpcui_message(port: int) -> None:
```

Comment:
I think this function should probably be renamed 😅

Reply:
what are your suggestions?

Reply:
start grpcui server?

Reply:
sgtm